packages/celotool/src/lib/geth.ts

```diff
@@ -536,7 +536,8 @@ export const unlock = async (
 export enum TestMode {
   Mixed = 'mixed',
   Data = 'data',
-  Transfer = 'transfer',
+  Transfer = 'celo_transfer',
```

Comment:
Do we know if this change is not going to break another script? I think that we could keep it as transfer just in case, and add in the description that 'transfer' translates to only celo.

Reply:
Good point, I reverted.